db/schema.rb

```diff
@@ -807,6 +810,9 @@
     t.boolean  "email_on_participation",           default: true,       null: false
     t.integer  "default_membership_volume",        default: 3,          null: false
     t.boolean  "has_muted",                        default: false,      null: false
+    t.string   "country"
```

Comment:
I also wonder if it might be cool to have users be able to specify (or at least view?) their location on their profile